tests/testthat/test-kwic.R

```diff
@@ -396,3 +396,32 @@ test_that("deprecated keywords argument still works", {
         "fox"
     )
 })
+
+test_that("test docvar is passed through kwic()", {
+    
+    # kwic only copy original docvars
+    testkwic <- kwic(data_corpus_inaugural, 'american')
+    expect_equal(attr(testkwic, 'docvars'),
```

Comment:
Unless users want to see docvars in concordance view, I don't think we need docvars.kwic(). I get docvars through as.tokens(), which multiplies rows of docvars.